src/ui/components/AppWrapper/AppWrapper.tsx

```diff
@@ -292,7 +305,7 @@ const AppWrapper = (props: { children: ReactNode }) => {
     );
   }
 
-  return initialised ? <>{props.children}</> : <></>;
```

Comment:
The reason we had this is so that we would know that we've connected to KERIA and loaded from the DB before we should the app. There is already an open PR for offline mode though

Reply:
So far during the seconds where the databases are loaded we do not show anything, rather a blank page.
Now a spinner is added to give the sensation of loading, besides this logic I have moved it to App.tsx where there are other auxiliary components like `LockModal`, `SetUserName`, `IncomingRequest`, `Settings`, `CustomToast`

Reply:
App.tsx

```
const renderApp = () => {
    if (!lockIsRendered && !stateCache.initialized) {
      // We need to include the LockModal in the loading page to track when is rendered
      return (
        <>
          <LockModal didEnter={() => setLockIsRendered(true)} />
          <div className="loading-page">
            <IonSpinner name="crescent" />
          </div>
        </>
      );
    } else if (showScan) {
      return <FullPageScanner setShowScan={setShowScan} />;
    } else {
      return (
        <>
          {isPreviewMode ? <MobileHeaderPreview /> : null}
          <Routes />
        </>
      );
    }
  };
```

Reply:
sounds good, missed the spinner thanks